types/porty/index.d.ts

```diff
@@ -0,0 +1,67 @@
+declare namespace Porty {
+    /**
+     * The host to run on
+     * @default "0.0.0.0"
+     */
+    export let HOST: string;
+    /**
+     * Minimum port
+     * @default 8000
+     */
+    export let MIN: number;
+    /**
+     * Maximum port
+     * @default 10000
+     */
+    export let MAX: number;
+
+    export interface FindOptions {
+        /** Ports to avoid */
+        avoids?: readonly number[] | undefined;
+        /**
+         * The host to run on.
+         * Defaults to {@link HOST}
+         */
+        host?: string | undefined;
```

Comment:
This is defined in the module scope but not accepted as an argument, AFAICT from the source code.

Reply:
Whoops, you're right. I guess I misread the code